### PR TITLE
Update compute add on docs for more clarity on disk throughput and iops

### PR DESCRIPTION
**Pull request by loong**
Changes:

* Some throughput values were slightly wrong, fixed those, used this [site](https://instances.vantage.sh/aws/ec2/m6g.16xlarge) for source of truth
* Moved disk-related values to their own table in their section
* Make sure to explicitly mention both throughput (Mbps) and IOPS instead of referring as Disk IO which is ambiguous

### CODE REVIEW

**Comment by kevcodez** [comment]
Could you please verify the numbers with @inian and if they are off, we need to adjust them in several more places (just search in frontend repo), stripe and backend. Otherwise we are introducing inconsistent numbers across the dashboard/website

**Comment by inian** [comment]
Woah looks like AWS changed these limits at some point. See [current](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-optimized.html) vs [past](https://web.archive.org/web/20230107213920/https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-optimized.html). 

I had [changed these](https://github.com/supabase/supabase/pull/11288) to match the aws docs then, but since they have updated it now, we should update on our side too

**Comment by loong** [comment]
ok will grep and see where else we need to update the values 👍

File: apps/docs/pages/guides/platform/compute-add-ons.mdx [hunk 1]
```diff
@@ -31,11 +33,32 @@ All Postgres instances on Supabase are dedicated applications running inside ded
...
+If you're unsure of how much throughput or IOPS your application requires, you can load test your project and inspect these [metrics in the Dashboard](https://app.supabase.com/project/_/reports). If the `Daily Disk IO Budget % Remaining` stat is less than 100%, it indicates that your workload has burst beyond the baseline IO throughput during the day. If this metric drops to zero, the workload has used up all the burst IO throughput minutes during the day and is running at the baseline performance. These projects are good candidates for upgrading to a larger compute add with higher baseline throughput.
```
Comment (gregnr) [inline apps/docs/pages/guides/platform/compute-add-ons.mdx:61]:
```suggestion
If you're unsure of how much throughput or IOPS your application requires, you can load test your project and inspect these [metrics in the Dashboard](https://app.supabase.com/project/_/reports). If the `Daily Disk IO Budget % Remaining` stat is less than 100%, it indicates that your workload has burst beyond the baseline IO throughput during the day. If this metric drops to zero, the workload has used up all the burst IO throughput minutes during the day and is running at the baseline performance. These projects are good candidates for upgrading to a larger compute add-on with higher baseline throughput.
```

**Review by gregnr — APPROVED** [review]

**Review by pcnc — APPROVED** [review]

**Pull request merged**